pypeit/spectrographs/keck_kcwi.py

```diff
@@ -115,13 +118,12 @@ def config_specific_par(self, scifile, inp_par=None):
         headarr = self.get_headarr(scifile)
 
         # Templates
-        print(self.get_meta_value(headarr, 'dispname'))
         if self.get_meta_value(headarr, 'dispname') == 'BH2':
             par['calibrations']['wavelengths']['method'] = 'full_template'  # 'full_template'
             par['calibrations']['wavelengths']['reid_arxiv'] = 'keck_kcwi_BH2_4200.fits'
             par['calibrations']['wavelengths']['lamps'] = ['FeI', 'ArI', 'ArII']
-            par['calibrations']['wavelengths']['n_first'] = 2
-            par['calibrations']['wavelengths']['n_final'] = 6
+            # par['calibrations']['wavelengths']['n_first'] = 4
```

Comment:
do you mean for these to be commented out?

Reply:
Yes - the default values work fine - removed